subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cipher.c

```diff
@@ -296,19 +295,19 @@ psa_status_t cracen_cipher_encrypt(const psa_key_attributes_t *attributes,
 			return PSA_ERROR_BUFFER_TOO_SMALL;
 		}
 		/* Handle inplace encryption by moving plaintext to the right by iv_length
-		* bytes. This is done because in inplace encryption the input and output
-		* should point to the same data so that the output can overwrite its own
-		* input. If they are not in sync the output will overwrite the input of
-		* another operation which is of course wrong.
-		*
-		*/
+		 * bytes. This is done because in inplace encryption the input and output
```

Comment:
Nit, you don't need to change it now since it is very annoying to do. But next time please do the space corrections etc in a separate commit so that they don't "hide" the actual content of the commit.